### ISSUE TITLE
After pulling I get:  The `--local-ip` flag is required for non-local Docker daemon

### ISSUE THREAD

**Issue by markrainer** [coreos/quayctl#58]
I'm running docker-machine which uses boot2docker.
docker-machine version 0.6.0, build e27fb87

docker -v
Docker version 1.10.0, build 590d5108

sha256:a3ed95caeb02 Completed  
sha256:f7c3a0dfba35 Completed  
sha256:25bdcd658780 Completed  
sha256:273c7f15da3b Completed  
sha256:4f2f3031c08c Completed  
sha256:82d5f57047e8 Completed  
sha256:cfd473635488 Completed  
sha256:4393485f2bb3 Completed  
sha256:1e14aab0083d Completed  
sha256:420890c9e918 Completed  
2016/06/15 15:44:46 The `--local-ip` flag is required for non-local Docker daemon

**Comment by josephschorr** [comment]
Hi Mark,

Yeah, you'll need to specify the IP address of your **host machine** as accessible inside the Docker VM via the `--local-ip` flag; this is so that your Docker VM can be loaded with the downloaded layers.

For my boot2docker install, the host machine is exposed as `10.0.2.2`, so I use `--local-ip=10.0.2.2`

**Comment by markrainer** [comment]
Hi Joseph,

docker-machine ls
NAME      ACTIVE   DRIVER       STATE     URL                         SWARM   DOCKER    ERRORS
soa-dev   \*        virtualbox   Running   tcp://192.168.99.100:2376           v1.11.2 

Looks like it pulls the image correctly, but just can't save it to my docker.

2016/06/15 22:56:40 Error pulling image into Docker: API error (500): Get https://192.168.99.100:5000/v1/_ping: dial tcp 192.168.99.100:5000: getsockopt: connection refused

I've also tried with the ip_address in the docker vm: 10.0.2.15 with the same results.

It's strange because when I first downloaded quayctl it didn't give me this message. At one point I blew away the whole VM just to try it again.

Anyway if you come up with anything let me know! Thanks again for your help!
